searx/searx
# Change logicodev dark infobox toggle color to dark

## What does this PR do?

Change background color and remove border of class `.infobox_toggle` for logicodev dark style.

## Why is this change important?

The current infobox toggle color is not dark for logicodev dark style. It should be dark color because the rest of the color theme is dark.

## How to test this PR locally?

- Go to preferences page of searx instance.
- Set themes to oscar and style to logicodev dark.
- Save changes.
- Search with keyword that will shows infobox, for example "Wikipedia".
- The toggle infobox should appear with dark color at the bottom of infobox.
- If the toggle infobox color is not dark, try to hard reload using <kbd>ctrl</kbd> + <kbd>F5</kbd> or <kbd>ctrl</kbd> + <kbd>shift</kbd> + <kbd>R</kbd>.

## Author's checklist

This is my first contribution in this project. Please tell me if I made a mistake or so :smile: 

When I tried to run `make themes` in local, all of the file header in directory `searx/static/themes/{oscar,simple}/{css,js}` is changed, even though I only changed `searx/static/themes/oscar/less/logicodev-dark/oscar.less` file. Is this intended?

Next thing is the changes in file `searx/static/themes/oscar/css/logicodev-dark.min.css` is generated because I made changes in `searx/static/themes/oscar/less/logicodev-dark/oscar.less`. But the changes didn't generated in file `searx/searx/static/themes/oscar/css/logicodev-dark.css`, instead the changes is generated in file `searx/static/themes/oscar/css/logicodev.css`.

I looked at `searx/static/themes/oscar/gruntfile.js` file and found this line.

https://github.com/searx/searx/blob/5fb9a5c7c10252b21a23fe66e1625ed9f8691b4a/searx/static/themes/oscar/gruntfile.js#L44-L45

Is this also intended?

The last weird things that also happened when I run `make themes` is file `searx/searx/static/css/bootstrap.min.css` is also changed. So, I just ignore it because I even didn't change anything in `searx/static/less/bootstrap` directory.

At the end, I decided to only commit the file that directly affected this patch. Tell me if I should commit some of those files too.

You can take a look all of the file changes when I run `make themes` in my repo branch at https://github.com/gagahpangeran/searx/commit/9159ea9824e70c163f556a240775203dac83fb2f

## Related issues

Closes #2410
